### PR TITLE
Fix/layout updates

### PR DESCRIPTION
**Pull request by zberwaldt**

### CODE REVIEW

File: components/BreadCrumb.tsx [hunk 1]
```diff
@@ -52,9 +51,14 @@ const Govcrumb: React.FC<GovcrumbProps> = ({ currentServerName }) => {
       }
     })
```
Comment (zberwaldt) [inline components/BreadCrumb.tsx:52]: Switch to use Memo per the associated card.

**Comment by zberwaldt** [comment]
This PR moves the poll listing to the associate server page. The layout was refactored to merge together the two pages. Additinoally some fixes for the breadcrumbs were added.

**Review by wpmonty — COMMENTED** [review]
Looks amazing! One small thing: the link to poll results just needs to append `polls/` otherwise its currently linking to a 404

File: pages/servers/[serverId]/polls/results/[pollId].tsx [hunk 2]
```diff
@@ -29,10 +29,20 @@ const PollResults: NextPage = () => {
...
       <Box bg='dark-1' maxW='2xl' mx='auto' p='2rem 3rem'>
-        <Govcrumb currentServerName={currentServer?.name}/>
+        {/* <Govcrumb currentServerName={currentServer?.name}/> */}
+        <Breadcrumb
+          spacing='8px'
+          separator={<ChevronRightIcon color='gray.500' />}
+          color='gray.300'
+        >
+          <BreadcrumbItem>
+            <BreadcrumbLink onClick={() => router.back()}>Back</BreadcrumbLink>
```
Comment (wpmonty) [inline pages/servers/[serverId]/polls/results/[pollId].tsx]: this should use root relative url

File: pages/servers/[serverId]/polls/results/[pollId].tsx [hunk 3]
```diff
@@ -27,10 +29,20 @@ const PollResults: NextPage = () => {
...
       <Box bg='dark-1' maxW='2xl' mx='auto' p='2rem 3rem'>
-        <Govcrumb />
+        {/* <Govcrumb currentServerName={currentServer?.name}/> */}
+        <Breadcrumb
+          spacing='8px'
+          separator={<ChevronRightIcon color='gray.500' />}
+          color='gray.300'
+        >
+          <BreadcrumbItem>
+            <BreadcrumbLink href={`/servers/${currentServer?.id}`}>Back</BreadcrumbLink>
```
Comment (wpmonty) [inline pages/servers/[serverId]/polls/results/[pollId].tsx]: Hmm optional chaining might return a weird route? Maybe a ternary might work more neatly. If there is no server ID, should just route to /servers

Comment (zberwaldt) [reply pages/servers/[serverId]/polls/results/[pollId].tsx]: I think we can do:

```suggestion
{`/servers/${currentServer?.id ?? ''}`}
```

Comment (zberwaldt) [reply pages/servers/[serverId]/polls/results/[pollId].tsx]: I'll fix it, regardless.

**Review by wpmonty — APPROVED** [review]

**Pull request merged**